app/controllers/site_controller.rb

```diff
@@ -1,4 +1,8 @@
 class SiteController < ApplicationController
   def index
   end
+
+  def search_gems
+  	@gems = GemsAdoption.search(params[:search])
+  end
```

Comment:
To make this more meaningful I think we should move the search logic to a ´searches_controller.rb´ and define this on the ´index´ method of that controller. What do you think @sebasoga @linatodi ?